### ISSUE TITLE
haproxy not updated

### ISSUE THREAD

**Issue by elmalto** [QubitProducts/bamboo#72]
I run the latest (as of today) bamboo version and run into the following issue:

- I update my marathon app (scaling or other changes, tried various combinations)
- Changes show up on http://localhost:8000/api/state
- haproxy does not get updated
- I receive 503 Service Unavailable
- I change ACL in UI and change it back
- haproxy is now updated

config:

```
{
  "Marathon": {
    "Endpoint": "http://localhost:8080"
  },

  "Bamboo": {
    "Endpoint": "http://10.218.146.232:8000",
    "Zookeeper": {
      "Host": "10.218.146.232:2181",
      "Path": "/marathon-haproxy/state",
      "ReportingDelay": 5
    }
  },

  "HAProxy": {
    "TemplatePath": "/opt/bamboo/config/haproxy_template.cfg",
    "OutputPath": "/etc/haproxy/haproxy.cfg",
    "ReloadCommand": "PIDS=`pidof haproxy`; haproxy -f /etc/haproxy/haproxy.cfg -p /var/run/haproxy.pid -sf $PIDS && while ps -p $PIDS; do sleep 0.2; done"
  },

  "StatsD": {
    "Enabled": false,
    "Host": "localhost:8125",
    "Prefix": "bamboo-server.development."
  }
}
```

**Comment by activars** [comment]
From what you described,  it seems that Bamboo didn't get scale events from
Marathon.

Did you enable marathon http callback?

On 7 Jan 2015, at 00:33, elmalto <notifications@github.com> wrote:

I run the latest (as of today) bamboo version and run into the following
issue:

   - I update my marathon app (scaling or other changes, tried various
   combinations)
   - Changes show up on http://localhost:8000/api/state
   - haproxy does not get updated
   - I receive 503 Service Unavailable
   - I change ACL in UI and change it back
   - haproxy is now updated

config:

{
  "Marathon": {
    "Endpoint": "http://localhost:8080"
  },

  "Bamboo": {
    "Endpoint": "http://10.218.146.232:8000",
    "Zookeeper": {
      "Host": "10.218.146.232:2181",
      "Path": "/marathon-haproxy/state",
      "ReportingDelay": 5
    }
  },

  "HAProxy": {
    "TemplatePath": "/opt/bamboo/config/haproxy_template.cfg",
    "OutputPath": "/etc/haproxy/haproxy.cfg",
    "ReloadCommand": "PIDS=`pidof haproxy`; haproxy -f
/etc/haproxy/haproxy.cfg -p /var/run/haproxy.pid -sf $PIDS && while ps
-p $PIDS; do sleep 0.2; done"
  },

  "StatsD": {
    "Enabled": false,
    "Host": "localhost:8125",
    "Prefix": "bamboo-server.development."
  }
}

—
Reply to this email directly or view it on GitHub
<https://github.com/QubitProducts/bamboo/issues/72>.

**Comment by elmalto** [comment]
Apparently disabled it when updating to marathon 0.7.7-Snapshot. Working now

**Comment by activars** [comment]
Thanks for the update.

Sent from my iPhone

On 7 Jan 2015, at 00:47, elmalto <notifications@github.com> wrote:

Closed #72 <https://github.com/QubitProducts/bamboo/issues/72>.

—
Reply to this email directly or view it on GitHub
<https://github.com/QubitProducts/bamboo/issues/72#event-215037063>.
